ruspy/estimation/estimation.py

```diff
@@ -42,6 +42,10 @@ def estimate(init_dict, df, maint_func=lin_cost, repl_4=False):
     states = df.loc[:, "state"].to_numpy()
     num_obs = df.shape[0]
     num_states = init_dict["states"]
+    if init_dict["maint_func"] == "linear":
```

Comment:
Why would we want this back in? Sth. Probably went wrong with your branching strategy. Please always check your diffs before requesting review.

Reply:
Rebased on master.

Reply:
Again learned something :)